quickstart/aws/tutorial-eks.md

```diff
@@ -108,24 +108,8 @@ In this tutorial, we'll launch a new Managed Kubernetes cluster in Elastic Conta
     +   │  ├─ aws:ec2:LaunchConfiguration             helloworld-nodeLaunchConfiguration  	create
     +   │  ├─ aws:cloudformation:Stack                helloworld-nodes                    	create
     +   │  └─ pulumi:providers:kubernetes             helloworld-provider                 	create
-    +   └─ aws-infra:network:Network                  vpc                               	create
-    +      ├─ aws:ec2:Vpc                             vpc                               	create
-    +      ├─ aws:ec2:Eip                             vpc-nat-0                         	create
-    +      ├─ aws:ec2:Eip                             vpc-nat-1                         	create
-    +      ├─ aws:ec2:InternetGateway                 vpc                               	create
-    +      ├─ aws:ec2:Subnet                          vpc-nat-1                         	create
-    +      ├─ aws:ec2:Subnet                          vpc-0                             	create
-    +      ├─ aws:ec2:Subnet                          vpc-nat-0                         	create
-    +      ├─ aws:ec2:Subnet                          vpc-1                             	create
-    +      ├─ aws:ec2:RouteTable                      vpc                               	create
-    +      ├─ aws:ec2:NatGateway                      vpc-nat-1                         	create
-    +      ├─ aws:ec2:RouteTableAssociation           vpc-nat-1                         	create
-    +      ├─ aws:ec2:NatGateway                      vpc-nat-0                         	create
-    +      ├─ aws:ec2:RouteTableAssociation           vpc-nat-0                         	create
-    +      ├─ aws:ec2:RouteTable                      vpc-nat-1                         	create
-    +      ├─ aws:ec2:RouteTable                      vpc-nat-0                         	create
-    +      ├─ aws:ec2:RouteTableAssociation           vpc-1                             	create
-    +      └─ aws:ec2:RouteTableAssociation           vpc-0                             	create
+    ...
```

Comment:
FYI, I just merged the change to convert the site from Jekyll to Hugo, which will cause a conflict here (sorry!). As part of that, I had to tweak these ```bash code snippets to workaround a markdown rendering issue with Hugo. For some reason, Hugo's markdown renderer does not like all the `+` in these code snippets.

To workaround, I removed the ```bash from these snippets, and instead just indented twice (8 spaces), which allows it to render as a code snippet (and render correctly). We lose the `bash` language, but that doesn't matter much for these particular snippets.